docs/wiki/framework/flash-framework.md

```diff
@@ -14,39 +14,57 @@ version:
 
 ## 1. Overview
 
-ACE provides a simple interface for creating flashbang grenades and specifying their properties.
+ACE provides a simple interface for creating flashbang grenades and specifying their properties. It is important that one sets `timeToLive` to be greater than the total possible time until the last explosion, i.e, `ace_grenades_flashbangBangs * ace_grenades_flashbangInterval + 3 * ace_grenades_flashbangIntervalSpread`. Any explosions that occur after the `timeToLive` has expired will occur at `(0,0)`.
 
 
 ## 2. Config Values
 
-There is only one config entry that is specific to ACE flashbangs: `ace_grenades_flashbang`. All successive values can be left undefined and they will be given the defaults shown in the first example below.
+There are several config entries specific to ACE flashbangs. All successive values can be left undefined and they will be given the defaults shown in the first example below.
 
 ```cpp
 class CfgAmmo {
     class MyDefaultFlashbang {
-        ace_grenades_flashbang[] = {0,0.25,0.05}; // Not actually a flashbang
-        //ace_grenades_flashbang[] = {NUMBER_OF_BANGS,BANG_INTERVAL,BANG_DEVIATION};
+        ace_grenades_flashbang = 0;                 // Not a flashbang
```

Comment:
But class is flashbang, would also make more sense since you continue comments with "with one bang".

Reply:
```suggestion
        ace_grenades_flashbang = 1;                 // A flashbang
```

Reply:
I wanted to list the defaults. I think I'll need to reword a little of the surrounding docs to change this.

Reply:
Just add: ` (default: 0 - not flashbang)`